ceph.spec.in

```diff
@@ -15,32 +15,32 @@
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 #
 %bcond_without ocf
-%if 0%{?suse_version}
-%bcond_with ceph_test_package
-%bcond_with cephfs_java
-%bcond_without lowmem_builder
-%else
-%bcond_without ceph_test_package
-%bcond_without cephfs_java
-%bcond_with lowmem_builder
-%endif
 %bcond_with make_check
 %ifarch s390 s390x
 %bcond_with tcmalloc
 %else
 %bcond_without tcmalloc
 %endif
 %if 0%{?fedora} || 0%{?rhel}
+%bcond_without lttng
```

Comment:
nit, might want to reorder `%bcond_without lttng` to put it right after `%bcond_with lowmem_builder` to match the order in `suse_version`.